modules/src/ics02_client/client_type.rs

```diff
@@ -1,8 +1,10 @@
 use super::error;
 use anomaly::fail;
+use serde_derive::{Deserialize, Serialize};
 
 /// Type of the consensus algorithm
-#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+
```

Comment:
Nitpick: attributes are typically on the very previous line to the item they annotate. Might be fixed by running rustfmt.

Reply:
fixed.. rustfmt didn't catch this.